.github/workflows/ci.yml

```diff
@@ -44,12 +44,7 @@ jobs:
       run: |
         go get -u github.com/vbatts/git-validation
         go get -u github.com/kunalkushwaha/ltag
-        go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
-        (
-          cd "${GOPATH}/src/github.com/golangci/golangci-lint/cmd/golangci-lint"
-          git checkout v1.23.8
-          go build -v && go install
-        )
+        GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.23.8
```

Comment:
This doesn't mess up the project's `go.mod`, correct?

Reply:
hmmm... it does modify go.mod and go.sum

Reply:
the other pattern seems to be using curl

https://github.com/containerd/release-tool/blob/3a655ed2950cb5e25f89d34d5c208193701d2475/.github/workflows/ci.yml#L44-L48

Reply:
Ok, so the trick is to run the `go get` from any directory outside of /go/src/github.com/containerd/console

Could be inside tempdir

Reply:
yup, so by not setting `working-directory:` in the "dependencies" step to the git checkout dir, this is actually working properly in this PR.